tests/testthat/test-multilevelannotationstep3.R

```diff
@@ -0,0 +1,36 @@
+patrick::with_parameters_test_that(
+  "multilevelannotation step 3 works",
+  {
+    testthat_wd <- getwd()
+    load("../../data/adduct_weights.rda")
+    
+    # load expected results
+    expected <- read.csv(file.path(getwd(), "testdata/advanced", subfolder, "Stage3.csv"))
+    expected$MatchCategory <- as.character(expected$MatchCategory)
+    
+    # set correct working directory
+    test_directory <- file.path(getwd(), "testdata/multilevelannotationstep3", subfolder)
+    setwd(test_directory)
+    
+    # load needed objects
+    load("stage2.Rda")
```

Comment:
I'm probably missing something, but I can't find this `stage2.Rda` file in the testdata. Could you give me a hint? Otherwise looks good!

Reply:
The test data are not present in the repository directly, but available on GitLab (commented in the PR description, the data currently in a PR on GitLab). The reason for this is minimize the size of the repository (as discussed generally [here](https://github.com/RECETOX/recetox-xMSannotator/pull/37#issuecomment-911789402)). Eventually an automatic script which downloads the test data can be done (when all tests are finished and test data unified).

Reply:
Yes, I read that, but I couldn't find the file in GitLab either, so I assume it has to be here. Sorry for not making it clear.

Reply:
Yeah, so far @xtrojak ran it with the files locally - I'll upload the files to the GitLab as complete set of testdata.

Reply:
@maximskorik now all test data is available!